### DISCUSSION TITLE
App timeout when deployed with Docker

### DISCUSSION THREAD

**Discussion by fmmattioni** [ThinkR-open/golem#586] [Q&A]
Hey everyone!

I have been recently using the app platform from digitalocean to deploy a golem app: https://www.digitalocean.com/products/app-platform/.

Deployment couldn't be easier: just set up a `.yaml` in the github repo, and they do the rest (recognize the `Dockerfile` in the repo, redeploy at every push, etc)...

One thing I noticed, though, is that when the app is deployed like this, it always times out (or grays out) after about 2 minutes of inactivity. Have any of you experienced something like this before? If so, what could be the most appropriate solution for this?

I am tempted to try this one out: https://stackoverflow.com/questions/54594781/how-to-prevent-a-shiny-app-from-being-grayed-out/54604662#54604662, but it feels a bit hacky than a definitive solution.

Thank you!
Felipe

**Comment by aftonsteps** [comment]
What does the .yaml file look like? I'm not sure how Digital Ocean works, but can you set deployment options such as timeout via a field in the .yaml file? The hacky solution looks like it could potentially fill up your logs with a lot of dots, so that would be inconvenient. 🙃

**Reply by fmmattioni** [reply]
That's a good point... I'm going to check whether I could set it.. these are the two yaml file templates that we use: https://github.com/digitalocean/sample-dockerfile/tree/main/.do
